### DISCUSSION TITLE
Unable to open "http://ip_address:2020" page

### DISCUSSION THREAD

**Discussion by Vlad-Kliucharov** [ludeknovy/jtl-reporter#94] [Q&A] [answered]
On my side, it's not working. I need help.
1) I had installed docker 
2) Run command "docker-compose up -d" 
3) Open browser add site address "http://IP_ADDRESS:2020" AND "site can’t be reached"
And one of my docker containers has the status "EXITED (0)"

![Screenshot_2](https://user-images.githubusercontent.com/20796574/140098693-d7fba0a7-2033-425c-bc20-e446141989a3.png)

**Comment by ludeknovy** [comment, marked as answer]
Hi @Vlad-Kliucharov, the exited migration container is supposed to be exited - it handles the db migrations. So no need to worry about it.

Did you try to access the application like this http://localhost:2020 ?

**Reply by Vlad-Kliucharov** [reply]
Hi @ludeknovy !
Thanks, now it works, cause in this case we use another item - "http://**localhost**:2020" in the URL, in the example on the site its looks like "http://**IP_ADDRESS**:2020"

**Reply by ludeknovy** [reply]
Yes, its on purpose, because if you run it locally, the IP is 127.0.0.1 / localhost. But if you run the app on a virtual machine, you access it through its IP address. But I might rephrase it, so its less confusing;)

Please, mark your question as answered. Thanks.

**Comment by Vlad-Kliucharov** [comment]
Please tell me how can I mark it as "answered"?

**Reply by ludeknovy** [reply]
see the green circle icon: https://github.com/ludeknovy/jtl-reporter/discussions/81

**Reply by Vlad-Kliucharov** [reply]
I have no option to make "answered", maybe you have any prompts? Beside of ticket

**Comment by jhmjam** [comment]
![Screenshot 2024-04-15 at 11 16 15](https://github.com/ludeknovy/jtl-reporter/assets/89923453/d8019917-7691-4bfe-9d46-e327b0e8151d)
Hi I used docker and ran the compose command but one of the containers failed.

Tried removing and started again but same result.

AM i doing something wrong?

**Reply by ludeknovy** [reply]
Hi @jhmjam can you share the docker compose logs? or just db logs? Without the logs, it's impossible to tell what could be the problem.

**Reply by jhmjam** [reply]
Hi i get permission denied for the docker logs but i have used docker login?

**Reply by ludeknovy** [reply]
Then you must have something wrong with your docker setup? You don't need docker login for that. 
My guess would be the permission issue is causing the container to fail, because it needs access to the file system to persist the data.

**Reply by jhmjam** [reply]
It works okay if i use docker desktop. I also tried sudo to get past it but no luck. Any suggestions?

**Reply by ludeknovy** [reply]
I guess the fix depends on your platform - try googling for docker compose permission issue for your platform.

**Reply by jhmjam** [reply]
Ok thanks :)
